rpc/block_svc_routines.c

```diff
@@ -3451,14 +3460,16 @@ block_create_cli_1_svc_st(blockCreateCli *blk, struct svc_req *rqstp)
     goto exist;
   }
 
+  cobj.prio_idx = blockGetPrioPathIndex(glfs, blk->volume, list);
```

Comment:
Shouldn't the prio calculations done based on whether the other servers in the cluster have this capability or not?

Reply:
That is way we fail if capabilities mismatch, hence the question above.